library/entities/Loan.java

```diff
@@ -1,79 +1,69 @@
 package library.entities;
+
 import java.io.Serializable;
 import java.text.SimpleDateFormat;
 import java.util.Date;
 
 @SuppressWarnings("serial")
 public class Loan implements Serializable {
-	
-	public static enum lOaN_sTaTe { CURRENT, OVER_DUE, DISCHARGED };
-	
-	private int LoAn_Id;
-	private Book BoOk;
-	private Member MeMbEr;
-	private Date DaTe;
-	private lOaN_sTaTe StAtE;
-
-	
-	public Loan(int loanId, Book bOoK, Member mEmBeR, Date DuE_dAtE) {
+
+	public static enum LoanState {
+		CURRENT, OVER_DUE, DISCHARGED
+	};
+
+	private int loanId;
+	private Book book;
+	private Member member;
+	private Date date;
+	private LoanState state;
+
+	public Loan(int loanId, Book book, Member member, Date DuE_dAtE) {
 		this.LoAn_Id = loanId;
-		this.BoOk = bOoK;
-		this.MeMbEr = mEmBeR;
+		this.book = book;
+		this.member = member;
 		this.DaTe = DuE_dAtE;

```

Comment:
Change the variable Name

Reply:
Thanks for the feedback, I will make the changes